docs/source/includes/nearest-neighbors/knn-introduction.rst

```diff
@@ -15,10 +15,10 @@
 .. *******************************************************************************/
 
 
-:math:`k`-NN :capterm:`classification` algorithm infers the class for the new
+:math:`k`-NN :capterm:`classification` and :capterm:`search` algorithm infers the class for the new
 feature vector by computing majority vote of the :math:`k` nearest observations
-from the training set.
-
+from the training set for :capterm:`classification` problem and search closest in
+distance for the new feature vector to the training set for :capterm:`search` problem.
```

Comment:
I'd suggest splitting this into two or three sentences. End the sentence after "feature vector" (line 19) and then describe how the class is inferred for each type of problem.

Reply:
I rewrote the sentences a bit. can you see?